### ISSUE TITLE
Add Contributing Guidelines

### ISSUE THREAD

**Issue by matthewsot** [UC-Davis-CS-Tutoring/Study-Guides#2]
This can just link to the CONTRIBUTING.md file in Athena once it is merged into master. See UC-Davis-CS-Tutoring/Athena#2

**Comment by matthewsot** [comment]
This would supersede the "Want to Contribute?" message in the README, and provide a central place for contributors to find guidelines.
